report/test/renderer/dom-test.js

```diff
@@ -94,6 +94,13 @@ describe('DOM', () => {
       assert.equal(result.innerHTML,
           '<a rel="noopener" target="_blank" href="https://example.com/foo"> Link </a> ' +
           'and some text afterwards.', 'link with spaces in brackets');
+
+      result = dom.convertMarkdownLinkSnippets(
```

Comment:
nit: maybe split this into a separate test so it's a little clearer that it's testing links with code snippets, but up to you